packages/verticaltabs/src/react/list.tsx

```diff
@@ -24,8 +24,9 @@ interface CollapsibleListProps extends ListProps {
   collapsed?: boolean
 }
 export const CollapsibleList: React.FC<CollapsibleListProps> = props => {
+  const { collapsed, ...rest } = props
```

Comment:
nit picky but why not just destructure above?

Reply:
personal preference. i don't like it when the line with the component definition breaks onto multiple lines - i think it makes it harder to read. also my editor highlighting can't always parse it. 🤷